datavault-assembly/src/datavault-home/bin/start-worker.sh

```diff
@@ -1 +1,13 @@
-# This is a wee shell script to strat the work, only I haven't written it yet....
\ No newline at end of file
+#!/bin/bash
+
+if [ -z "$DATAVAULT_HOME" ]; then
+    echo "DATAVAULT_HOME is not set"
+    exit 1
+fi
+
+cd ${DATAVAULT_HOME}/lib
```

Comment:
Given that it happens in its own shell does it matter? However, I've made a mess of this PR so I'm going to cancel it anyway and we can do this later. Cheers